Source/Fuse.Alerts/Triggers.uno

```diff
@@ -29,28 +29,29 @@ namespace Fuse
 	public delegate void AlertHandler(object sender, AlertArgs args);
 
 	/**
-		This is trigger action for showing native alert dialog with a single button. Only available on iOS or Android
+This is trigger action for showing native alert dialog with a single button. Only available on iOS or Android
```

Comment:
This comment and other comments should be indented inside the `/**` with two tabs like before